Zend/zend_attributes.c

```diff
@@ -121,7 +121,8 @@ ZEND_API int zend_get_attribute_value(zval *ret, zend_attribute *attr, uint32_t
 		return FAILURE;
 	}
 
-	ZVAL_COPY_OR_DUP(ret, &attr->argv[i]);
+	// TODO: Named args.
```

Comment:
nit: I'd assume this TODO is outdated based on Zend/tests/named_params/attributes_positional_after_named.phpt passing.

Reply:
There is one part here that is not quite done yet, I've added some extra tests and moved the TODO in https://github.com/php/php-src/pull/5357/commits/30701fa47bee8df77d27c7e6460452dcb8712c60. This is a more general problem, e.g. we also don't perform the usual "too many arguments" check for internal attributes right now.